CHANGES.md

```diff
@@ -16,6 +16,10 @@ None or N/A.
 
 ### Enhancements
 
+[#636](https://github.com/cylc/cylc-ui/pull/636) -
+Improve the appearence of mutation menus and display an expandable short list
```

Comment:
s/appearence/appearance ? (Not sure if this spelling exists too in us/gb english)

Reply:
no, it is an error